modules/fscloud/main.tf

```diff
@@ -1,16 +1,24 @@
 module "event_notification" {
-  source                        = "../../"
-  resource_group_id             = var.resource_group_id
-  name                          = var.name
-  plan                          = "standard"
-  kms_encryption_enabled        = true
-  skip_iam_authorization_policy = var.skip_iam_authorization_policy
-  existing_kms_instance_crn     = var.existing_kms_instance_crn
-  root_key_id                   = var.root_key_id
-  kms_endpoint_url              = var.kms_endpoint_url
-  tags                          = var.tags
-  service_endpoints             = "public-and-private"
-  cbr_rules                     = var.cbr_rules
-  region                        = var.region
-  service_credential_names      = var.service_credential_names
+  source                    = "../../"
+  resource_group_id         = var.resource_group_id
+  name                      = var.name
+  plan                      = "standard"
+  kms_encryption_enabled    = true
+  skip_en_kms_auth_policy   = var.skip_en_kms_auth_policy
+  existing_kms_instance_crn = var.existing_kms_instance_crn
+  root_key_id               = var.root_key_id
+  kms_endpoint_url          = var.kms_endpoint_url
+  tags                      = var.tags
+  service_endpoints         = "public-and-private"
+  cbr_rules                 = var.cbr_rules
+  region                    = var.region
+  service_credential_names  = var.service_credential_names
+  # COS Related
+  cos_integration_enabled = var.cos_integration_enabled
+  cos_endpoint            = var.cos_endpoint
+  cos_destination_name    = var.cos_destination_name
+  cos_bucket_name         = var.cos_bucket_name
+  cos_instance_id         = var.cos_instance_id
+  cos_region              = var.cos_region
+  skip_en_cos_auth_policy = var.skip_en_cos_auth_policy
```

Comment:
please update the usage block in the fscloud submodule readme with the new variables